ajax/libs/ityped/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "author": "Luis Vinicius <luisviniciusbarreto@gmail.com>",
+  "description": "Dead simple animated typing, with no dependencies.",
+  "filename": "index.min.js",
+  "homepage": "https://ityped.surge.sh/",
+  "keywords": [
+    "typed",
+    "ityped",
+    "letters",
+    "animation",
+    "typing",
+    "animated typing"
```

Comment:
Use `animated` is enough because when you type "animated typing", line 11 would also match it.